extensions/amazon-lambda-http/deployment/src/main/java/io/quarkus/amazon/lambda/http/deployment/AmazonLambdaHttpProcessor.java

```diff
@@ -60,4 +63,23 @@ void setTempDir(BuildProducer<SystemPropertyBuildItem> systemProperty) {
         systemProperty.produce(new SystemPropertyBuildItem(FileResolver.CACHE_DIR_BASE_PROP_NAME, "/tmp"));
     }
 
+    @BuildStep
+    public void generateScripts(OutputTargetBuildItem target,
+            BuildProducer<ArtifactResultBuildItem> artifactResultProducer) throws Exception {
+        String lambdaName = LambdaUtil.artifactToLambda(target.getBaseName());
+
+        String output = LambdaUtil.copyResource("lambda/bootstrap-example.sh");
+        LambdaUtil.writeFile(target, "bootstrap-example.sh", output);
```

Comment:
I can't help thinking that we need a build item to represent these additional files, so we have some way of tracking all the outputs of the build. I don't actually have a use case for this at the moment and this is fine for now, but maybe something we should revisit later.